codespell_lib/data/dictionary.txt

```diff
@@ -9404,6 +9404,10 @@ defautls->defaults
 defautlt->default
 defautly->defaultly
```

Comment:
@luzpaz  your PR seem fine. This one is not your PR, but do you mind changing this one?
`defaultly` is an invalid word in reputable major dictionaries.
Shall we suggest  `default` instead of `defaultly` ?

```text
defautly->defaultly
```

Reply:
Done, [bad57b5](https://github.com/codespell-project/codespell/pull/2395/commits/bad57b5c8edee95a441780bd7988b4f73f392084). Thanks!